src/bindings/js/node/tests/unit/tensor.test.js

```diff
@@ -252,6 +252,58 @@ describe("ov.Tensor tests", () => {
       assert.strictEqual(tensor.getSize(), expectedSize);
     });
   });
+  describe("BigInt Tensor support", () => {
```

Comment:
I double-checked, and we've already tested with big int arrays for Tensor.
Please remove this block.